instructionAPI/src/Instruction.C

```diff
@@ -558,13 +558,24 @@ namespace Dyninst
                 case e_sysexit:
                 case e_call:
                 case e_syscall:
-                case amdgpu_op_s_setpc_b64:
-                case amdgpu_op_s_swappc_b64:
-                case amdgpu_gfx908_op_S_SETPC_B64:
-                case amdgpu_gfx908_op_S_SWAPPC_B64:
-                case amdgpu_gfx90a_op_S_SETPC_B64:
-                case amdgpu_gfx90a_op_S_SWAPPC_B64:
                     return false;
+#if 1
```

Comment:
I'm assuming this was for debugging? Let's remove it.

Reply:
This code sequence is a temporary until real issues are fixed.
By #if the sequence that will be removed when further work is done,
this is completely identified as to what will  be removed.